tests/functional/spec/relationships.spec.js

```diff
@@ -191,6 +192,72 @@ describe('Relationships', function() {
     });
   });
 
+  it('Given I am completing the survey, When I have added one or more household members after answering the relationships question, Then I will be asked for the original household members relationships to all new members', function() {
```

Comment:
Not specific to this PR, but the given should probably be in the initial describe call. e.g. `describe('Given I am completing the test_relationships survey')`

Reply:
You could make this section a bit clearer by nesting these tests under another describe saying that they've added three people.